gradle/libs.versions.toml

```diff
@@ -1,11 +1,11 @@
 [versions]
-android-compile = "33"
+android-compile = "34"
```

Comment:
This is bumping the target SDK as well?

If so, have you checked that Kable properly handles API 34?

Or, all changes necessary for API 34 are covered by this PR?

Reply:
Looked into it, it does **not** bump the target SDK version.
Target defaults to **min** SDK version (see https://github.com/JuulLabs/sensortag/pull/90 for more details).

_For a library, the target SDK doesn't matter much, as the app's manifest will override the target version anyways, see [this Reddit comment](https://www.reddit.com/r/androiddev/comments/46ab5u/should_libraries_define_targetsdkversion/d03w7yd/?utm_source=share&utm_medium=web3x&utm_name=web3xcss&utm_term=1&utm_content=share_button) for more details._